docs/index.yaml

```diff
@@ -2,21 +2,21 @@ apiVersion: v1
 entries:
   splunk-connect-for-kubernetes:
   - apiVersion: v1
-    appVersion: 1.5.4
+    appVersion: 1.5.4-opendoor.1
```

Comment:
I'd prefer staying with `1.5.5` or `1.6.0`, samething for subcharts.

this chart is end of support so we don't have to worry about upstream releasing new versions